src/style.css

```diff
@@ -0,0 +1,121 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  --white: #fff;
+  --blue: #00acdc;
+  --navy-blue: #060b35;
+}
+
+.card {
+  padding-top: 32px;
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  background: var(--white);
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__img {
+  margin-left: 19px;
+  margin-bottom: 40px;
+  width: 160px;
+  height: 134px;
+}
+
+.card__title {
+  margin: 0 16px 4px;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--navy-blue);
+}
+
+.card__product-code {
+  margin-left: 16px;
+  margin-bottom: 16px;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+}
+
+.card__reviews {
+  display: flex;
+  justify-content: space-between;
+  margin: 0 16px 24px;
+}
+
+.card__reviews-text {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--navy-blue);
+}
+
+.stars {
```

Comment:
You should create separate file for this: https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md [11 point]